docs/Features/XMLFeatures.md

```diff
@@ -2,6 +2,10 @@
 
 [vscode-xml](https://github.com/redhat-developer/vscode-xml) provides a number of notable features, with demos below.
 
+## Formatting
+
+For formatting capabilities, see [Formatting](../Formatting).
```

Comment:
Link should be ../Formatting.md#formatting

Reply:
To be consistent with other feature i will détail more the content

* ctrl shift k to format
* a little demo which indents xml

I will end with
* see formatting for more information